templates/moinmoin/files/setup/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+MOINVERSION=1.9.10
+
+# Download MoinMoin
+curl -o moin-install.tar.gz http://static.moinmo.in/files/moin-$MOINVERSION.tar.gz
+tar xvzf moin-install.tar.gz
+mv moin-$MOINVERSION moin-install
+
+# Apply patches
+patch moin-install/MoinMoin/logfile/editlog.py ~/setup/editlog.patch
+patch moin-install/MoinMoin/logfile/eventlog.py ~/setup/eventlog.patch
+
+# Move install to app_dir
+cp -a ~/moin-install/* /app/
+
+# Remove the install
+rm -rf ~/moin-install
+rm ~/moin-install.tar.gz
+
+# Stage files that will go to mounts on deploy
```

Comment:
For clarify, edit comment that it's staging files that will get copied to mounts on *first* deploy.

Reply:
Also, nitpick: Comments should end in a period.